bitnami/mongodb/values.yaml

```
@@ -1104,3 +1148,311 @@ metrics:
     ##       summary: High request latency
     ##
     rules: {}
+
+##
+## MongoDB(R) Hidden node parameters.
+##
+hidden:
+  ## Enable deploying the MongoDB(R) Hidden Node
+  ##   https://docs.mongodb.com/manual/tutorial/configure-a-hidden-replica-set-member/
+  ##
+  enabled: false
+
+  ## MongoDB(R) configuration file for Primary and Secondary nodes. For documentation of all options, see:
+  ##   http://docs.mongodb.org/manual/reference/configuration-options/
+  ##
+  configuration: ""
+
+  ## ConfigMap with MongoDB(R) configuration for Primary and Secondary nodes
+  ## NOTE: When it's set the arbiter.configuration parameter is ignored
+  ##
+  # existingConfigmap:
```

Review these details (result of copy-pasting):
```suggestion
  ## MongoDB(R) configuration file for Hidden nodes. For documentation of all options, see:
  ##   http://docs.mongodb.org/manual/reference/configuration-options/
  ##
  configuration: ""

  ## ConfigMap with MongoDB(R) configuration for Hidden nodes
  ## NOTE: When it's set the hidden.configuration parameter is ignored
  ##
  # existingConfigmap:
```